kernel/transaction/type/xa/core/pom.xml

```diff
@@ -53,6 +53,11 @@
             <groupId>com.mysql</groupId>
             <artifactId>mysql-connector-j</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
+            <artifactId>ojdbc8</artifactId>
```

Comment:
- The LICENSE of `com.oracle.database.jdbc:ojdbc8` is not allowed by ASF, please change it to,
```xml
<dependency>
            <groupId>com.oracle.database.jdbc</groupId>
            <artifactId>ojdbc8</artifactId>
            <version>${ojdbc8.version}</version>
            <scope>test</scope>
            <optional>true</optional>
</dependency>
```
- Consider https://lists.apache.org/thread/rqmq3t0b31fg3cz7jsm80rzpk18srnts and https://www.apache.org/legal/resolved.html for explanations of this.